src/App.css

```diff
@@ -1,3 +1,55 @@
 .App {
   text-align: center;
 }
+
+.navbar {
+  background-color: #333;
+  overflow: hidden;
+}
+.navbar a {
+  float: left;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+.navbar a:hover {
+  background-color: #ddd;
+  color: black;
+}
+.navbar a.active {
+  background-color: #04aa6d;
+  color: white;
+}
+
+.imgIMG {
+  max-width: 100%;
+  height: auto;
+  box-shadow: 0 0 5px 0 rgb(103, 103, 105);
+}
+
+.listDisplay {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+.divDisplay {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 20%;
+  padding: 10px;
+}
+@media screen and (max-width: 800px) {
+  .divDisplay {
+    flex: 50%;
+    max-width: 40%;
+  }
+}
+@media screen and (max-width: 600px) {
+  .divDisplay {
+    flex: 100%;
+    max-width: 100%;
+  }
+}
```

Comment:
Really good work on this ticket, Marek. Well implemented media queries for responsive screen size and a well thought out display. Great job.